apps/web/src/components/SearchBar/SearchBar.scss

```diff
@@ -0,0 +1,35 @@
+.searchbar-container {
+  position: relative;
+}
+
+input {
+  width: 100%;
+  padding: 0.625rem 3rem;
+  border: 1px solid white;
+  border-radius: 5rem;
+}
+
+input::placeholder,
+input {
+  font-size: 1rem;
+}
+
+input:hover,
+input:focus {
+  outline: 2px solid #3478cc;
+}
+
+input:focus {
+  border-radius: 30px 30px 0px 0px;
+}
+
+.searchbar-container::after {
+  content: url('../../assets/searchIcon.svg');
```

Comment:
Just something to think about:

What if we had the requirement that the `searchIcon` was a different color based on `:focus` state? Like it should be rendered in a lighter color if it's not focused?

Reply:
I could not figure this out. I tried adding an opacity of 0.75 to the icons, and :hover an opacity of 1, but this did not work.

Reply:
Good things to try for sure. Same thing that I did when I had to do this for work. 

I'll put the detailed solution into Slack. Don't let me forget.

Reply:
I noticed the input loses its outline (hover state) when you hover over the search icon. It seems like the hover outline should be maintained. I'm not convinced this is the best solution, but you could use the `.searchbar-container` div for the outline/border instead of the input.

Reply:
I think if you're using html elements (`span`) for the search and close icons, you don't need to use the pseudo elements (::before and ::after). As I understand it, their whole function is to add styling/content to a page without adding html to the DOM. I see in another resolved conversation that the pseudo elements weren't interactive because they aren't actually in the DOM which is why you used spans so totally understandable, but it might make sense to refactor to a different approach now that we know those pseudo-elements don't suit this situation. Maybe Will J. can chime in if I'm off base here.